model/src/main/kotlin/utils/PurlExtensions.kt

```diff
@@ -145,3 +148,13 @@ fun String.toProvenance(): Provenance {
         else -> UnknownProvenance
     }
 }
+
+@Suppress("SwallowedException")
+fun String.toIdentifier(): Identifier {
```

Comment:
Please add documentation to this public method.
Currently you need to know the file name and the function name to understand that this method is about converting a _purl_ String to an identifier.